pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -112,6 +112,9 @@ const (
 	// PodAutoscalerConditionReady is set when the revision is starting to materialize
 	// runtime resources, and becomes true when those resources are ready.
 	PodAutoscalerConditionReady = apis.ConditionReady
+	// PodAutoscalerConditionHasBeenActive is set when the PodAutoscaler's ScaleTargetRef was ready to
+	// serve traffic once.
+	PodAutoscalerConditionHasBeenActive apis.ConditionType = "HasBeenActive"
```

Comment:
`HasBeenActive` sounds a bit odd to be - how about `Deployed`

Reply:
`TargetDeployed` might be more specific

Reply:
I almost feel like we should just set `Ready` :joy:

Reply:
Not necessarily true. With initialScale=0 — nothing will ever be deployed

Reply:
Hmm `ScaleTargetDeployed` might imply new calculated autoscaler scale targets. What about `InitialTargetDeployed`?

Reply:
> Not necessarily true. With initialScale=0 — nothing will ever be deployed

but we expect this condition to be 'true' when initial scale = 0

Reply:
yes.

Reply:
`TargetInitialized` ?

Reply:
Or maybe ReachedInitialScale, since that’s what actually gates the condition

Reply:
`ScaleTargetInitialized` would be my favorite mix from the above. I would like to keep  InitialScale out of this, it doesn't necessarily gate this, for example if minScale is > initialScale.

Reply:
Hm, in my head when minScale > initialScale, initialScale is implicitly set to minScale, but I see how that could be confusing.

`ScaleTargetInitialized` seems alright, just a bit odd that it talks about a "Scale Target" concept that isn't really defined anywhere else, and doesn't mention the quite important fact that it's the scale target for the _first_ scale we're talking about. I'm also not totally sure what "Initialized" means in this context.. "-Reached" seems clearer about what has to happen. Not the end of the world, though, if everyone else is happy Im happy.

Reply:
I think it works because it's part of the PodAutoscaler spec

https://github.com/knative/serving/blob/f43143931a69a23ba9b2ddddff4b190494e72ca2/pkg/apis/autoscaling/v1alpha1/pa_types.go#L98-L100